examples/agones-xonotic-sidecar/sidecar.yaml

```diff
@@ -42,7 +42,7 @@ spec:
               image: gcr.io/agones-images/xonotic-example:0.8
             - name: quilkin
               image: us-docker.pkg.dev/quilkin/release/quilkin:0.4.0
-              args: ["run", "--port", "26001", "--to", "127.0.0.1:26000"]
+              args: ["proxy", "--port", "26001", "--to", "127.0.0.1:26000"]
```

Comment:
Can we take `us-docker.pkg.dev/quilkin/release/quilkin:0.4.0` ➡ `us-docker.pkg.dev/quilkin/release/quilkin:0.5.0`, since the new syntax won't work with 0.4.0?